src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,41 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMAL_AGE = 18;
+    private static final int MINIMAL_PASSWORD_LENGTH = 6;
+    private StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getLogin() == null) {
+            throw new RuntimeException("Your login can not be null. Please, enter a valid login");
+        } else if (user.getLogin().isEmpty()) {
+            throw new RuntimeException("Your login can not be empty. Please, enter a valid login");
+        } else if (user.getLogin().isBlank()) {
+            throw new RuntimeException("Your login can not contain blank spaces only."
+                    + " Please, enter a valid login");
```

Comment:
Let create one if statement for invalid login, too much lines
```suggestion
        if (user.getLogin() == null || (user.getLogin().isEmpty() || user.getLogin().isBlank()) {
            throw new RuntimeException("Please, enter a valid login");
        }
```